maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+use Swagger\Client\CurationCMS\Api\TopicsApi;
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\Swagger\ApiProvider;
+use \Wikia\Logger\WikiaLogger;
+
+ini_set('display_errors', 'stderr');
+ini_set('error_reporting', E_ALL);
+
+require_once(dirname(__FILE__) . '/../../Maintenance.php');
+
+class updateCurationCMSTopicRanks extends Maintenance {
+
+	const SERVICE_NAME = "curation-cms";
+	const TIMEOUT = 60;
+	const NO_CONTENT = 204;
+
+	const SUCCESS_MESSAGE = "Updated topic ranks in the Curation CMS";
+	const EXCEPTION_MESSAGE = "An unknown error occurred while updating topic ranks";
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Trigger the Curation CMS to update its Topic rankings";
+	}
+
+	public function execute() {
+		$api = $this->getTopicsApi();
+		try {
+			$startTime = microtime( true );
+			$api->updateRank();
+			$this->logSuccess( $startTime );
+		} catch ( Exception $e ) {
+			$this->logException( $e );
+		}
+	}
+
+	private function logSuccess( $startTime ) {
+		WikiaLogger::instance()->info( self::SUCCESS_MESSAGE,
+			[
+				'elapsedTime' => microtime( true ) - $startTime
+			]
+		);
+	}
+
+	private function logException( Exception $e ) {
+		WikiaLogger::instance()->error( self::EXCEPTION_MESSAGE,
+			[
+				'exception_message' => $e->getMessage(),
```

Comment:
Its standard indentation.  Plus, doesn't it look a little familiar? ;)

https://github.com/Wikia/app/blame/a66ef23f66883fa9d5cf6f1283a2a44111873a6a/maintenance/wikia/cronjobs/deleteOldOnSiteNotifications.php#L57-L63